hydra-node/test/Hydra/Chain/DirectSpec.hs

```diff
@@ -3,58 +3,68 @@
 -- | Integration tests for the Direct chain component which does interact with a
 -- single mocked "node". For tests spinning up real cardano-nodes see
 -- 'EndToEndSpec'.
+--
+-- XXX(AB): Are those tests really useful? What are they really testing? It seems to me
+-- we need either to turn those into proper Unit tests, eg. not requiring a full-blown mock server to run,
+-- or drop them altogether in favor of `DirectChainSpec` tests which test the real thing.
```

Comment:
I'd say they integration test the off-chain behavior of our `Chain.Direct`, i.e. everything without ledger + plutus validation. This overlaps greatly with `DirectChainSpec` but also with `TxSpec` and we might be able to make them more complementar.